src/server/system_services/pool_server.js

```diff
@@ -136,7 +145,8 @@ async function _init() {
                     update: {
                         pools: [{
                             _id: pool._id,
-                            'hosts_pool_info.initialized': true
+                            $set: { 'hosts_pool_info.initialized': true },
+                            $unset: { 'hosts_pool_info.init_timeout': 1 }
```

Comment:
Just a suggestion: In my opinion using the value true for $unset is more readable and less confusing than using the value 1